Core/Core.cpp

```diff
@@ -335,7 +306,8 @@ bool Core_Run(GraphicsContext *ctx) {
 		if (GetUIState() != UISTATE_INGAME) {
 			Core_StateProcessed();
 			if (GetUIState() == UISTATE_EXIT) {
-				UpdateRunLoop();
+				// Not sure why we do a final frame here?
```

Comment:
I'm not either.  I actually hacked up a way to run non-x86 jit from my PC (to be able to easily iterate before testing on device), and it surprised me when I crashed exiting the game (which I never started, because the jit isn't valid for x86...)

Was added in ae19c4813866b4da597a8f237a3b4ceb8824060c, probably some shutdown fix...

-[Unknown]

Reply:
Right.. I'll look into getting rid of it again later.